### DISCUSSION TITLE
Projects (beta): boolean statements in filter syntax

### DISCUSSION THREAD

**Discussion by noamross** [community/community#5533] [Projects and Issues]
I am not sure if this exists and is undocumented, or is unimplemented, but it would be good to be able to use some boolean syntax (OR statements and maybe parentheses nesting) to filter issues in the Project views.  For instance, I may want to view projects that have a status of `To Do` or `In Progress` but not `Done`

**Comment by noamross** [comment]
Ah, I had not realized that this work with the syntax `status:"To Do","In Progress"`, though more complex booleans may not work.

**Reply by astout** [reply]
Hmm, I want logical OR on separate fields. i.e. For my team, we have many repos for one product. So we have a "product" custom "Single Select" field. I would like to have a Projects (Beta) view that shows all issues where `product:"product name" OR repo:product-backend,product-frontend` etc. I can't find a way to do this currently.

**Reply by brandonlenz** [reply]
Similar desire from my current team ([posted below](https://github.com/github-community/community/discussions/5533#discussioncomment-2948163))

**Comment by brandonlenz** [comment]
There's still a need for the logical OR, in my opinion. For example, for sprint planning, it's really handy to be able to see either the `@previous` and `@current` iterations or `@current` and `@next` iterations (depending on the timing of your sprint planning activity) _alongside_ the backlog, aka `No iteration`.

That might look something like 
- `no:sprint OR sprint:@previous,@current,@next`
- `no:sprint || sprint:@previous,@current,@next`

**Reply by brandonlenz** [reply]
If you were to try either of those filters (without some logical OR syntax) right now, you'd get no results because the filter uses a logical AND by default between separate filters, while comma separations in a single filter use logical OR.

If anyone knows of another workaround for such a search, let me know!

**Comment by commondatageek** [comment]
My use case for this today:

I would like to be able to see a list of all items that are either:
- assigned to me — `assignee:commondatageek`
- OR assigned to nobody (so I can see new issues I can pick up) — `no:assignee`

I can't figure out how to make this work.  I have tried the following:

- `assignee:commondatageek,""`
- `assignee:commondatageek no:assignee`
- and many other variations, etc etc.

I also tried one where I exclude my teammates:

`-assignee:jake,andrew`

However, that one ALSO excluded any issues that were assigned to BOTH myself and one of my teammates, which was not helpful.
